### PR TITLE
Add mutator and accessor functions for some layers

### PR DESCRIPTION
**Pull request by sreenikSS**
1. Add accessor and mutator functions for the padding dimensions of the Convolution Layer.
2. Add accessor function for obtaining the input and output dimensions of the LinearNoBias layer.

### CODE REVIEW

File: src/mlpack/methods/ann/layer/atrous_convolution.hpp [hunk 1]
```diff
@@ -205,7 +205,7 @@ class AtrousConvolution
   //! Modify the output height.
   size_t& OutputHeight() { return outputHeight; }
 
-  //! Get the input size
+  //! Get the input size.
```
Comment (zoq) [inline src/mlpack/methods/ann/layer/atrous_convolution.hpp]: Nice catch!

Comment (sreenikSS) [reply src/mlpack/methods/ann/layer/atrous_convolution.hpp]: Haha, was pointed out by @walragatver in a previous discussion.

**Comment by sreenikSS** [comment]
@zoq thanks for approving.

I'd be happy if I can get an opinion regarding something I've been thinking about. I might need a few other changes in the mlpack code to make it fully compatible for the model converter.

In that case I would have to open another PR (and there would be multiple closed PRs doing the same thing). Instead, would you suggest I make the changes in my branch and `make install` that branch in some folder of my local machine and put my include path to that folder when I work on the model converter, and then at the end of work (when no more changes are required) push the changes to this one PR?

To summarize, is opening multiple PRs doing small parts of one big chunk of work considered good practice?

**Comment by rcurtin** [comment]
> To summarize, is opening multiple PRs doing small parts of one big chunk of work considered good practice?

Up to you---it makes it easier to review and get it merged in, but it's also a little more overhead for you.  Personally I'm fine with either approach; not sure if others have different opinions.

**Comment by zoq** [comment]
Agreed, it makes the review process easier, if you are going for multiple PR's it would be great if you could layout the things you like to change, so that we can get an overview.
